src/Block/Renderer/DocumentRenderer.php

```diff
@@ -21,9 +21,9 @@
 class DocumentRenderer implements BlockRendererInterface
 {
     /**
-     * @param Document $block
+     * @param AbstractBlock|Document $block
+     * @param HtmlRendererInterface $htmlRenderer
      * @param bool $inTightList
-     *
```

Comment:
Actually nvm, I shouldn't be asking you to make more changes simply to please my inner OCD, especially when your improvements are great as-is.